src/state/slices/plotSlice.test.tsx

```diff
@@ -0,0 +1,39 @@
+import PlotReducer, { initialState, createPlot, closePlot } from './plotSlice';
+
+describe('plotSlice', () => {
+  // only test the hard to test bits of columnSlice here - like column reordering logic
```

Comment:
Probably worth removing this to save confusion down the line

Reply:
oops, my copy-pasting has been exposed 🙈